website/content/api-docs/secret/pki.mdx

```diff
@@ -1136,13 +1137,42 @@ $ curl \
 #### Sample Response
 
 ```json
+
```

Comment:
nit: spare line in JSON body.

Reply:
Fixed, bad merge it seems, thanks for catching it.